aten/src/ATen/native/cuda/DilatedMaxPool2d.cu

```diff
@@ -401,9 +441,7 @@ void max_pool2d_with_indices_out_cuda_template(
     }
   );
 
-  TORCH_CHECK(cudaGetLastError() == cudaSuccess,
-     "max_pool2d_with_indices_out_cuda_frame failed with error code ",
-     cudaGetLastError());
+  THCudaCheck(cudaGetLastError()); 
```

Comment:
Nit: can you use AT_CUDA_CHECK here, not to propagate use of old THC apis? Also, similar incorrect use of cudaGetLastError() is in other pooling implementations (fractional, average) - can you please fix it too?

Reply:
Sure, I will replace them with AT_CUDA_CHECK.